cmd/httpclient/main.go

```diff
@@ -15,12 +15,12 @@
 //
 // Examples:
 //
-//   ./httpclient -dns-transport system ...
-//   ./httpclient -dns-transport godns ...
-//   ./httpclient -dns-transport doh ...
-//   ./httpclient -dns-transport dot ...
-//   ./httpclient -dns-transport tcp ...
-//   ./httpclient -dns-transport udp [-dns-udp-server <addr>:<port>] ...
+//   ./httpclient -dns-server system:// ...
+//   ./httpclient -dns-server godns:// ...
+//   ./httpclient -dns-server https://<url> ...
```

Comment:
Would it be better to replace <url> with <urn>?

Reply:
Not sure, but I have two observations that probably allow us to dodge the question :-P. They are:

1. can you use `system:///` and `godns:///` rather than `system://` and `godns://`, because the former are canonical URLs (the final `/` is the path) while the latter are not?

2. my initial examples here were too terse. Can you change them such that if one copies and pastes the command line it works? This entails using here, and in the `-help` output, really working DNS servers like the `dnsclient` command already does?